.github/workflows/jacoco_check.yml

```diff
@@ -51,24 +50,38 @@ jobs:
           java-version: "adopt@1.8"
       - name: Build and run tests
         run: sbt ++${{matrix.scala}} jacoco -DSPARK_VERSION=${{matrix.spark}}
-      - name: Add coverage to PR
+      - name: Add coverage of 'cobol-parser' to PR
+        id: jacocoParser
+        uses: madrapps/jacoco-report@v1.3
+        with:
+          paths: >
+            ${{ github.workspace }}/cobol-parser/target/scala-${{ matrix.scalaShort }}/jacoco/report/jacoco.xml
+          token: ${{ secrets.GITHUB_TOKEN }}
+          min-coverage-overall: ${{ matrix.overall }}
+          min-coverage-changed-files: ${{ matrix.cobolParserOverride }}
+          title: JaCoCo code coverage report - 'cobol-parser'
```

Comment:
```suggestion
          title: JaCoCo code coverage report - 'cobol-parser' - scala:${{ matrix.scala }} - spark:${{ matrix.spark }}
```

Reply:
I disagree - the Scala and Spark version is irrelevant to the code coverage. Including it is confusing.

Reply:
I agree with you. My idea is from practise that different scala version leads to different report content. So keep this information should be useful for later orientation what was the input conditions to reach these numbers.

Reply:
Until we understand why coverage reporting for different Scala versions differs when the code base is the same, I consider this a Jacoco issue, not Cobrix issue. 

The purpose of these reports is to help us understand the coverage of Cobrix, not for debugging Jacoco issues.

Reply:
Sure, because on Cobrix is only one Scala version running in PR, there is not needed to keep this information in title.